PhotoPlusFrontend/PhotoPlus/src/app/components/products/products.component.ts

```diff
@@ -43,9 +46,23 @@ export class ProductsComponent implements OnInit {
     });
   }
 
+
+  buy(product: Product) {
+    this.cartService.clearCart();
```

Comment:
We cant destroy all of user's picks just because they want to buy sth immediately or even just cause they misclick